tests/garage/test_dtypes.py

```diff
@@ -2,9 +2,13 @@
 import numpy as np
 import pytest
 
+from garage import Sample
+from garage import SamplesBatch
 from garage import TrajectoryBatch
 from garage.envs import EnvSpec
 
+# ===========================TrajectoryBatch-Test=========================== #
```

Comment:
please don't leave useless block comments like this

Reply:
if you want to group tests, put them in different files or use a class within the same file.

Reply:
removed this and similar block comments